scripts/build/generateIcons.ts

```diff
@@ -180,6 +194,22 @@ class GeneratedFilesManifest {
     }
 }
 
-generate().catch(error => {
-    console.error('Failed to generate icons: %s', (error as Error).message)
-})
+async function loadCodiconMappings(): Promise<Record<string, number | undefined>> {
```

Comment:
We can delete this if `src/codewhisperer/views/css/securityPanel.css` is updated to not refer to `content` directly